docs/dashboard/customers.md

```diff
@@ -63,6 +63,11 @@ In the General Information card, there is a checkbox next to the text _User&nbsp
 
 If you have implemented a backend protocol that requires users to confirm account activation from an e-mail, which is not a default feature, this may also be useful if a user has issues with replying to the e-mail and asks for a manual activation.
 
+### How to issue a gift card
+Once the customer has been created in the dashboard, users can issue a gift card. Click the *Issue new card* button of the Gift Cards section in the Customer Overview card. Then fill in all the information required for a specific gift card and click the *Issue* button. You can issue multiple gift cards to the customer.
```

Comment:
1. `Once the customer has been created in the dashboard, users can issue a gift card` Can't users create gift cards before they create customers? 
2.  `Customer Overview card` - which card is that? cannot see it on the screen
3.  `You can issue multiple gift cards to the customer` - that's not proper english, is it?
4.  `You can issue multiple gift cards to the customer` -> from this view?

Reply:
1. This section is about customers. If you want to issue a gift card from the customer details view, first, you need to create a customer, am I right? But FYI, I've removed that sentence, and it will be updated with the next commit. 
2. I've changed it to the customer details view. It will be replaced with the next commit. 
3/4. It's grammatically correct, but it can be written in a different way. Anyway, that sentence will be removed with the next commit.

Reply:
Ok, just ping me to check the final version once it's pushed to repo 👍